runner/utils.go

```diff
@@ -25,12 +25,14 @@ func isTmpDir(path string) bool {
 
 func isIgnoredFolder(path string) bool {
 	paths := strings.Split(path, "/")
-	if len(paths) <= 0 {
+	pathsOnWindows := strings.Split(path, "\\")
```

Comment:
I think you are running that example on the go playground, and since it runs on unix, it splits paths with the unix separator. if you run it on windows, it will split with the windows separator.